content/attacks/posts/2018-06-20-Bithumb.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2018-06-20
+categories: "Custodian"
+title: "Bithumb Hacked for $31 Million"
+---
+
+## Summary
+
+On June 20, 2018, [Bithumb, a South Korean cryptocurrency exchange, was hacked for $31.5 million](https://www.investopedia.com/news/crypto-exchange-bithumb-hacked-loses-315m/). The attacker gained access to one of the exchange's hot wallets, which are used to store user funds that are available for withdrawal.
+
+## Attackers
+
+The identity of the attacker(s) is unknown.
+
+## Losses
+
+Bithumb previously announced that the amount of damage caused by the cyberattack was about 35 billion won ($31.5 million). However, 7 days after the hack, it was revealed that the [actual damage was around 19 billion won ($17.1 million) instead](https://cafe.bithumb.com/view/board-contents/1033767).
```

Comment:
Write only actual funds lost, and move the last sentence to the Timeline section